plugins/jobs/src/js/JobStopRun.js

```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import { componentFromStream } from "data-service";
+import { stopJobRun } from "#SRC/js/events/MetronomeClient";
+import PropTypes from "prop-types";
+
+import { Subject } from "rxjs/Subject";
+import "rxjs/add/operator/map";
+import "rxjs/add/operator/switchMap";
+import "rxjs/add/operator/do";
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/startWith";
+import "rxjs/add/operator/catch";
+
+import JobStopRunModal from "./components/JobStopRunModal";
+
+function executeStopRequest({ jobID, jobRun, onSuccess }) {
+  return stopJobRun(jobID, jobRun)
+    .map(_ => ({ done: true }))
```

Comment:
We could use `mapTo` here http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html#instance-method-mapTo and spare a function

Reply:
Nice :D.